docs/pages/workflows/variables.mdx

```diff
@@ -12,36 +12,57 @@ jobs:
   fingerprint:
     outputs:
       # @info Sets the <CODE>fingerprint_hash_for_workflow</CODE> as a variable #
-      fingerprint_hash_for_workflow: ${{ steps.fingerprint_step_id.outputs.fingerprint_hash }}
+      fingerprint_hash_for_workflow: ${{ steps.fingerprint_hash_step.outputs.fingerprint_hash_for_workflow }}
       # @end #
     steps:
-      - name: Set fingerprint hash
-        id: fingerprint_step_id
-        # @info Exposes the <CODE>fingerprint_hash</CODE> variable to other steps in this job #
-        outputs: [fingerprint_hash]
+      - uses: eas/checkout
+      - uses: eas/install_node_modules
+      - name: Install additional tools
+        run: sudo apt-get update -y && sudo apt-get install -y jq
+      - name: Set fingerprint variables
+        id: fingerprint_hash_step
+        # @info Exposes the <CODE>fingerprint_hash_for_workflow</CODE> and <CODE>fingerprint_hash_for_steps</CODE> variables. #
+        outputs: [fingerprint_hash_for_workflow, fingerprint_hash_for_steps]
         # @end #
-        # @info Sets the <CODE>fingerprint_hash</CODE> as a variable #
+        # @info Sets the <CODE>fingerprint_hash_for_workflow</CODE> and <CODE>fingerprint_hash_for_steps</CODE> variables #
         run: |
-          FINGERPRINT_HASH=$(npx expo-updates fingerprint:generate --platform ios | jq '.hash')
-          set-output fingerprint_hash $FINGERPRINT_HASH
+          FINGERPRINT=$(npx expo-updates fingerprint:generate --platform ios)
```

Comment:
this needs to do excludes based on workflow in order to generate consistent hashes across environments (before and after pod install for managed workflows for example)